frontend/vue/components/UtilityPanel/UtilityPanel.vue

```diff
@@ -167,17 +175,29 @@ export default class UtilityPanel extends Vue.with(Props) {
     top: 6rem;
   }
 
+  &__header {
+    flex: 0 0 auto;
+  }
+
+  &__content {
+    flex: 1 0 0;
+    overflow: auto;
+    padding-bottom: 8rem;
```

Comment:
what are your thoughts of using the nearest Carbon value: `$spacing-12` (6 rem) or `$spacing-13` (10 rem). 

I don't think the design had a specific requirement here, we maybe this space is more just for leaving room for the Cookie preferences tab at the bottom?

Reply:
I didnt know about the 12 or 13, it sounds good, I will take a look and decide what is best.